src/gettingstarted/cli.md

```diff
@@ -75,6 +75,54 @@ Options:
  --shell (-s)          Launch the shell
 ```
 
+### CLI features
+
+Additional settings to control the operation of the `platform` CLI can be managed in the configuration file (See `.platform/local/project.yaml`) or environment variables. See the [`platformsh-cli` `README` for details](https://github.com/platformsh/platformsh-cli/blob/master/README.md#usage). 
```

Comment:
Additional settings to control the operation of the Platform.sh CLI can be managed in the configuration file ( `.platform/local/project.yaml`) or environment variables. See the [README](https://github.com/platformsh/platformsh-cli/blob/master/README.md#usage) for the CLI for more details.